test/runnable/template4.d

```diff
@@ -430,44 +429,6 @@ void test13()
 
 /*********************************************************/
 
-template zebra(string w)
-{
-    static if (w.length > 2 && w[1] == 'q')
-        const bool zebra = true;
-    else
-        const bool zebra = false;
-}
-
-template horse(string w)
-{
-    static if (w.length == 1 || w[1] == 'q')
-        const bool horse = true;
-    else
-        const bool horse = false;
-}
-
-void test14()
-{
-    bool lion = zebra!("a");
-    writeln(lion);
-    assert(!lion);
-    lion = zebra!("aqb");
-    writeln(lion);
-    assert(lion);
-
-    lion = horse!("a");
-    writeln(lion);
-    assert(lion);
-    lion = horse!("aqb");
-    writeln(lion);
-    assert(lion);
-    lion = horse!("ab");
-    writeln(lion);
```

Comment:
I don't really know why we need `writeln` here, except perhaps debugging... Anyway `printf` will do